libsolidity/inlineasm/AsmData.h

```diff
@@ -67,7 +67,7 @@ struct VariableDeclaration { SourceLocation location; TypedNameList variables; s
 /// Block that creates a scope (frees declared stack variables)
 struct Block { SourceLocation location; std::vector<Statement> statements; };
 /// Function definition ("function f(a, b) -> (d, e) { ... }")
-struct FunctionDefinition { SourceLocation location; std::string name; TypedNameList arguments; TypedNameList returns; Block body; };
+struct FunctionDefinition { SourceLocation location; std::string name; TypedNameList parameters; TypedNameList returns; Block body; };
```

Comment:
Can you also rename `returns` to `returnParameters`?

Reply:
I am not really sure about this one. A variable that holds the return value is not really a parameter. Perhaps `returnVariables`, `resultVariables` or short: `returns` or `results`?

Reply:
`returnVariables` or `outVariables`?